### ISSUE TITLE
500 error upon onepage checkout initialization with invalid API details

### ISSUE THREAD

**Issue by JayShore** [MtGox/magento#8]
The module throws the entire onepage checkout into a 500 error and causes shutdown of the Zend engine via incorrect exception handling within the checkout module. Should be linked with depends Mage_Sales and Mage_Payment and change exception behavior to integrate Zend and Mage exceptions to mitigate failure of the entire web framework.

**Comment by JayShore** [comment]
FYI - This is also referenced by issue #5. 

Apache error log: PHP Fatal error: Class 'Zend_Log' not found in /app/code/community/Mtgox/Bitcoin/Model/Bitcoin.php

Fix: webroot/index.php line 59 becomes "error_reporting( E_COMPILE_ERROR|E_RECOVERABLE_ERROR|E_ERROR|E_CORE_ERROR );"

Not a "true" fix. I see your commit comment @ jgautheron, I'll patch my installation and see what happens although I've already got it working temporarily.

**Comment by dp2r** [comment]
I have updated the files but still receive the fatal error re Zend_Log not found...etc.

So I thought I would start over. I created a new install of Magento v1.7.0.2 then installed this mod and the exact same error occurs. Even with the mod disabled the error still happens on checkout.

I created several new api keys with all possible options and tried each one, clearing cache between each and still the same problem occurs.

JayShore, how did you get it working?
